### PR TITLE
Adds `.avarc` meta file support.

### PR DESCRIPTION
**Pull request by jtag05**
Simple implementation resolving #520.

### CODE REVIEW

**Comment by novemberborn** [comment]
Hey @jtag05, thanks for the PR. #520 has already been closed as wontfix. Clearly this is a contentious issue but I'd like to see an argument in favor of placing configuration outside of `package.json` that goes beyond "it's what I'm used to from other tools", which I think is where we left #520.

Anyway, #520 is the place for that discussion, though of course the "it's what I'm used to" point has already been made.

**Comment by pho3nixf1re** [comment]
@novemberborn there is a perfectly valid argument in #520 that was recently posted. The package.json file has become a dumping ground for configuration. In large projects it makes it huge and unwieldy. Digging through a giant JSON file is not a pleasant experience and is reminiscent of the Java world's XML configs. My team prefers the dotfiles as it neatly separates each tool's configuration that is easily explorable. It is not just because "we are used to it". Every other tool we use manages to support this trivial feature that we value but AVA. It seems absurd that even in the face of community outcry and the presence of a perfectly fine PR that the maintainers here ignore the need.

**Comment by jtag05** [comment]
I'd like to see an argument against implementing 7 lines of code that has no performance issues. I understand that preferences are exactly that, but rather than taking the stance of "why should we?", as a utility should you not be approaching this from the perspective of "Why not?" as a means of maximizing adoption?

I think it's fair to argue that whichever route a developer chooses (meta file or package.json) all configuration should be stored the same way. As I have previously stated in the #520 thread, I was a supporter of `package.json` based configutation, but I've since run into issues on projects where a `package.json` file get's seriously bloated. Take for instance if you use a custom `eslintConfig` and store it in `package.json`, that can easily run 300 lines.

File: profile.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const EventEmitter = require('events');
 const meow = require('meow');
 const Promise = require('bluebird');
 const pkgConf = require('pkg-conf');
+const fs = require('pkg-conf');
```
Comment (tacooper) [inline profile.js]: I think you meant `require('fs')` here.

**Comment by psirenny** [comment]
To add to what @pho3nixf1re said, I prefer dotfiles because they can easily be added to a .npmignore file. Consumers of my packages don't care about various config choices and don't need to download bloated package.json file.

**Comment by novemberborn** [comment]
@psirenny the latest beta release (tagged `next` on npm) has support for `ava.config.js` files. Not a dotfile but you can keep config out of `package.json` and away from npm.

**Pull request closed without merging**